frontend/src/components/FormatNumber.jsx

```diff
@@ -0,0 +1,5 @@
+const FormatNumber = (hours) => {
```

Comment:
This could be named renderFormattedNumber and include the other checks that are done in render.